package.json

```diff
@@ -91,6 +100,7 @@
     "karma-rollup-preprocessor": "github:jlmakes/karma-rollup-preprocessor#master",
     "karma-safari-launcher": "1.0.0",
     "less": "4.1.2",
+    "log4js": "6.4.1",
```

Comment:
Why is this needed? If it's only for telling when karma config is invalid, I don't think it's noteworthy to add a new dependency for it.

Reply:
It was already a Karma dependency, so it's not any additional weight. But we could run the file through `require()` and prevalidate it. Then we wouldn't need the dep.

Reply:
Nah let it be then.

Reply:
@Gusted Background of the change was, karma was throwing a "syntax error in karma.conf.js" error that was never making it to my terminal because it was getting log level from the config.

But I just looked at Karma source. Looks like they provide a way to pass level in. So we can have our cake and eat it too.